lib/internal/streams/operators.js

```diff
@@ -153,25 +178,26 @@ function map(fn, options) {
     try {
       while (true) {
         while (queue.length > 0) {
-          const val = await queue[0];
+          const { 0: value, 1: error } = await queue[0];
```

Comment:
why?

Reply:
If we add a catch handler to the value put into the queue then we don't have a problem.

Reply:
Already tried that, it doesn't work

Reply:
Which test fails?